Assets/MixedRealityToolkit.SDK/Experimental/NoneNativeKeyboard/Scripts/UICollection.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// This component represents and ordered collection of UI elements. You
+    /// can add to the UICollection by either dropping UI elements in the UI
+    /// element this component is attached to in the Editor or by calling the
+    /// AddItem(RectTransform) method. UI elements that are added to this
+    /// collection via the Editor will be automatically arranged when this
+    /// component executes.
+    /// To use this component attach it to a UI element (a GameObject with a
+    /// RectTransform component) such as an Image or Panel.
+    /// </summary>
+    [AddComponentMenu("UI/HoloUIKit/UICollection")]
```

Comment:
Suggest moving this into the and MRTK/Experimental menu

Reply:
@thalbern  Let's move this into mrtk experimental menu

Reply:
good catch - we'll adjust in follow up PR

Reply:
Any reason we cannot address it in this PR? Is it because you're not able to modify the branch since Alexees owns it and would need to give you write permissions?

Reply:
yeah i didn't have permission. - fixed the entry now in https://github.com/HoloSpaces/MixedRealityToolkit-Unity/pull/2 - we can merge that PR into this one